recipes/magnum/all/conanfile.py

```diff
@@ -0,0 +1,710 @@
+        "target_desktop_gles": [True, False],
+        "target_headless": [True, False],
+        "target_vk": [True, False],
+
+        "with_audio": [True, False],
+        "with_debugtools": [True, False],
+        "with_gl": [True, False],
+        "with_meshtools": [True, False],
+        "with_primitives": [True, False],
+        "with_scenegraph": [True, False],
+        "with_shaders": [True, False],
+        #"with_shaderstools": [True, False],  Option not available in sources!
+        "with_text": [True, False],
+        "with_texturetools": [True, False],
+        "with_trade": [True, False],
+        "with_vk": [True, False],
+        
+        "with_androidapplication": [True, False],
+        "with_emscriptenapplication": [True, False],
+        "with_glfwapplication": [True, False],
+        "with_glxapplication": [True, False],
+        "with_sdl2application": [True, False],
+        "with_xeglapplication": [True, False],
+        "with_windowlesscglapplication": [True, False],
+        "with_windowlesseglapplication": [True, False],
+        "with_windowlessglxapplication": [True, False],
+        "with_windowlessiosapplication": [True, False],
+        "with_windowlesswglapplication": [True, False],
+        "with_windowlesswindowseglapplication": [True, False],
+
+        "with_cglcontext": [True, False],
+        "with_eglcontext": [True, False],
+        "with_glxcontext": [True, False],
+        "with_wglcontext": [True, False],
+
+        "with_gl_info": [True, False],
+        # "with_vk_info": [True, False], Not in sources
+        "with_al_info": [True, False],
+        "with_distancefieldconverter": [True, False],
+        "with_fontconverter": [True, False],
+        "with_imageconverter": [True, False],
+        "with_sceneconverter": [True, False],
+        # "with_shaderconverter": [True, False],  Not in sources
+
+        # Options related to plugins
+        "with_anyaudioimporter": [True, False],
+        "with_anyimageconverter": [True, False],
+        "with_anyimageimporter": [True, False],
+        "with_anysceneconverter": [True, False],
+        "with_anysceneimporter": [True, False],
+        #"with_anyshaderconverter": [True, False],  Not in sources
+        "with_magnumfont": [True, False],
+        "with_magnumfontconverter": [True, False],
+        "with_objimporter": [True, False],
+        "with_tgaimporter": [True, False],
+        "with_tgaimageconverter": [True, False],
+        "with_wavaudioimporter": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "shared_plugins": True,
+
+        "target_gl": True,
+        "target_gles": True, # TODO: Here we probably have a CHOICE OPTION
+        "target_gles2": True, # TODO: Here we probably have a CHOICE OPTION
+        "target_desktop_gles": True, # TODO: Here we probably have a CHOICE OPTION
+        "target_headless": True,
+        "target_vk": True,
+
+        "with_audio": False,
```

Comment:
We mostly use the `with_` prefix when an options causes an external dependency to be pulled in.
I would rename these to `audio` or `enable_audio`.